libs/core/src/lib/radio/radio-button/radio-button.component.ts

```diff
@@ -34,6 +34,37 @@ export class RadioButtonComponent implements OnChanges, AfterViewInit, CssClassB
     @ViewChild('inputElement')
     inputElement: ElementRef;
 
+    /**
+     * Attached to the aria-label attribute of the host element. In most cases, aria-labelledby will
+     * take precedence so this may be omitted.
+     */
+    @Input('aria-label')
+    ariaLabel = '';
```

Comment:
The same comment for ariaLabels like on the other places

Reply:
Done